CONTRIBUTING.md

```diff
@@ -1,19 +1,87 @@
+<!-- markdownlint-disable MD041 -->
+[Français](#comment-contribuer)
+<!-- markdownlint-enable MD041 -->
+
 # How to Contribute
 
-When contributing, post comments and discuss changes you wish to make via Issues.
+When contributing, post comments and discuss changes you wish to make via Issues, while respecting our [code of conduct](CODE_OF_CONDUCT.md).
 
 Feel free to propose changes by creating Pull Requests. If you don't have write access, editing a file will create a Fork of this project for you to save your proposed changes to. Submitting a change to a file will write it to a new Branch in your Fork, so you can send a Pull Request.
 
 If this is your first time contributing on GitHub, don't worry! Let us know if you have any questions.
 
+## General changes
+
+All proposed changes will go through two levels of review:
+
+1. Automated checks (e.g., markdownlint, link checks)
```

Comment:
Might need some additional guidance here, or maybe look at common rules people will trip on. Also/alternately start adding child `.markdownlint.json` if stuff like the views have/need different rules